sources/packages/backend/workflow/test/test-utils/mock/mock.utils.ts

```diff
@@ -159,13 +175,24 @@ export function createWorkersMockedData(
       mockedData = mockedData[subprocessId];
     });
     // Create result mocked object.
+    const isMultiInstance = !!mockedWorker.options.multiInstanceLoopCounter;
+    let mockedDataValue: unknown = mockedData;
+    if (isMultiInstance) {
+      mockedData[IS_MULTI_INSTANCE] = isMultiInstance;
+      if (!mockedData[INSTANCES]) {
+        mockedData[INSTANCES] = [];
+      }
```

Comment:
There is minor suggestion from sonar here. 
![image](https://github.com/user-attachments/assets/2e3de5de-4222-4f29-9d72-8094dc163e2d)